mod/host.py

```diff
@@ -2360,8 +2360,10 @@ def snapshot_remove(self, idx):
         if idx < 0 or idx >= len(self.pedalboard_snapshots) or self.pedalboard_snapshots[idx] is None:
             return False
 
+        snapshot_to_remove = self.pedalboard_snapshots[idx]
         self.pedalboard_modified = True
-        self.pedalboard_snapshots[idx] = None
+        self.pedalboard_snapshots.remove(snapshot_to_remove)
+        logging.info(self.pedalboard_snapshots)
```

Comment:
this logging is good for testing, but not in production.
change it to `logging.debug` please